edx_solutions_projects/management/commands/revert_rename_projects_app.py

```diff
@@ -2,9 +2,10 @@
 Management command to revert renaming of projects app
 """
 import logging
-from south.db import db
+
 from django.core.management.base import BaseCommand
 from django.db import transaction
+from south.db import db
```

Comment:
We should update this old syntax to `from django.db import connection` to execute query